charts/marge-bot/values.yaml

```diff
@@ -40,3 +40,5 @@ secret:
   # yours: base64 -w0 <my_private_key>
   sshKey: ""
   authToken: ""
+  rebaseRemotely: false
```

Comment:
Are you sure boolean values are supported in marge-bot ?
Seems to use configargparse.ArgParser, don't know if it supports this

Reply:
right don't knwo either but I guess string is safer